LUIS/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+// This is a template webpack config file with minimal configuration.
+// Users are free to create their own webpack configuration files in their application.
+const webpack = require('webpack');
+const path = require('path');
+module.exports = {
+  entry: './lib/luisAuthoring.ts',
+  devtool: 'source-map',
+  output: {
+    filename: 'luisAuthoringBundle.js',
+    path: __dirname,
+    libraryTarget: 'var',
+    library: 'luisAuthoring'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
+        exclude: /(node_modules|test)/
+      }
+    ]
+  },
+  // "ms-rest-js" and "ms-rest-azure-js" are dependencies of this library.
+  // Customer is expected to import/include this library in browser javascript
+  // (probably using the script tag in their html file).
+  externals: {
+    "ms-rest-js": "msRest",
```

Comment:
Part of the rationale for not using AutoRest in the beginning was the heavy dependencies it introduces for trivial  http calls. Library size will be greatly affected - going from just a dozen bytes or so to a few hundred. This will give some users pause when considering it as a library.